crates/sui-framework/src/lib.rs

```diff
@@ -79,6 +80,23 @@ pub enum EventType {
     User,
 }
 
+// TODO: remove thse in favor of new costs
```

Comment:
```suggestion
// TODO: remove these in favor of new costs
```

Also, when are the "new costs" become available. Is this something that is currently in the works on the core Move side?

Reply:
Yes it's already kinda ready. We just need to convert them into the new logic.
The costs were calibrated with old logic in mind.